pkg/types/validation/installconfig.go

```diff
@@ -229,3 +233,39 @@ func validatePlatform(platform *types.Platform, fldPath *field.Path, openStackVa
 	}
 	return allErrs
 }
+
+func validateProxy(p *types.Proxy, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if p.HTTPProxy == "" && p.HTTPSProxy == "" {
+		allErrs = append(allErrs, field.Required(fldPath, "Must include httpProxy or httpsProxy."))
+	}
+	if p.HTTPProxy != "" {
+		if err := validate.URI(p.HTTPProxy); err != nil {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("HTTPProxy"), p.HTTPProxy, err.Error()))
+		}
+		if strings.HasPrefix(p.HTTPProxy, "https://") {
```

Comment:
This should be `!strings.HasPrefix(p.HTTPProxy", "http://")` so that it fails validation when the scheme is anything other than http.